docs/user/manifests/kubeapps-local-dev-auth-proxy-values.yaml

```diff
@@ -8,3 +8,5 @@ authProxy:
     - -cookie-secure=false
     - -oidc-issuer-url=https://172.18.0.2:32000
     - -ssl-insecure-skip-verify=true
+    # If you need to access the actual token in the frontend for testing, uncomment the following.
+    # - -set-authorization-header=true
```

Comment:
@vrabbi Sorry, I told you `--pass-authorization-header=true` but that passes it upstream, rather than this one above which sets it in the response back to the browser, so you can see the bearer token.